transactions/stx.ts

```diff
@@ -194,8 +191,6 @@ export async function generateUnsignedStxTokenTransferTransaction(
       memo,
       sponsored,
       anchorMode,
-      postConditions,
-      postConditionMode,
```

Comment:
we should probably just remove these from the outer function signature as well, while we're in here, and refactor the code which calls it.